.github/actions/kfp-cluster/action.yml

```diff
@@ -36,11 +40,23 @@ runs:
           ./.github/resources/scripts/build-images.sh
         fi
 
-    - name: Deploy KFP
+    - name: Deploy KFP with Database
       shell: bash
+      if: ${{ inputs.pipeline_store == 'database'}}
       run: |
         if [ "${{ inputs.proxy }}" = "true" ]; then
           ./.github/resources/scripts/deploy-kfp.sh --proxy
         else
           ./.github/resources/scripts/deploy-kfp.sh
         fi
+
+
+    - name: Deploy KFP with K8s Native API
```

Comment:
Optional: It seems like we don't need to have a separate step and we could just do like how `inputs.proxy` is used.

Reply:
I'm surprised this code is still there, because I removed these lines. I'll take a look again.